--- a/tools/build/src/engine/build.sh
+++ b/tools/build/src/engine/build.sh
@@ -132,7 +132,7 @@
 
     gcc)
         # Check whether it's MinGW GCC, which has Windows headers and none of POSIX ones.
-        machine=$(gcc -dumpmachine 2>/dev/null)
+        machine=`gcc -dumpmachine 2>/dev/null`
         if [ $? -ne 0 ]; then
             echo "BOOST_JAM_TOOLSET is gcc, but the 'gcc' command cannot be executed."
             echo "Make sure 'gcc' is in PATH, or use a different toolset."
